crates/ika-core/src/dwallet_mpc/mpc_manager.rs

```diff
@@ -391,77 +391,88 @@ impl DWalletMPCManager {
     /// Returns the completed computation results.
     pub(crate) async fn perform_cryptographic_computation(
         &mut self,
+        last_read_consensus_round: u64,
     ) -> HashMap<ComputationId, DwalletMPCResult<mpc::GuaranteedOutputDeliveryRoundResult>> {
         let mut ready_to_advance_sessions: Vec<_> = self
             .mpc_sessions
             .iter()
-            .filter(|(_, session)| session.status == MPCSessionStatus::Active)
             .filter_map(|(_, session)| {
+                if !matches!(session.status, MPCSessionStatus::Active { .. }) {
+                    return None;
+                }
+
                 // Only sessions with MPC event data should be advanced
-                session.mpc_event_data.clone().and_then(|mpc_event_data| {
+                session.request_data.clone().and_then(|request_data| {
```

Comment:
It's not what we talked about, we said we should take it from the `Active` status, and its not an option